internal/rbd/rbd_util.go

```diff
@@ -1171,11 +1171,21 @@ func genVolFromVolumeOptions(
 		rbdVol.NamePrefix = namePrefix
 	}
 
-	rbdVol.Monitors, rbdVol.ClusterID, err = util.GetMonsAndClusterID(volOptions)
-	if err != nil {
-		log.ErrorLog(ctx, "failed getting mons (%s)", err)
+	if checkClusterIDMapping {
```

Comment:
@Rakshith-R just wondering why we need this hard check in place?

Reply:
@humblec ,without this check, we may end up using mapped clusterid during create volume request, since we're using this function in parsecreatevolreq too.